pypeit/core/arc.py

```diff
@@ -729,6 +831,9 @@ def detect_lines(censpec, sigdetect = 5.0, input_thresh = None, cont_subtract=Tr
     max_frac_fwhm:  float, default = 2.5
        maximum width allowed for usable arc lines expressed relative to the fwhm.
 
+    min_pkdist_frac_fwhm: float, default = 0.75
```

Comment:
I'm not sure how to do that. Please provide instructions.

Reply:
@kbwestfall -- Please advise.

Reply:
```
cd doc
make clean ; make html
open _build/html/index.html
```

Reply:
I just checked this (pypeit/doc/_build/html/pypeit.core.html) and pypeit.core.arc isn't even displayed, which likely means that the sphinx choked on the whole file.

Generally, I don't know how `min_pkdist_frac_fwhm: float, default = 0.75` would be displayed, but it doesn't follow the nominal syntax.  Should be:

```
    min_pkdist_frac_fwhm (float, optional):
        Default is 0.75.
```

But I don't think it's worth including the default in the description because people can see it in the argument list, and it saves you having to change it in the description when it changes in the argument list.

I suggest we merge without fixing this because it's part of a much larger documentation issue.